ovos_PHAL_plugin_system/__init__.py

```diff
@@ -273,7 +302,31 @@ def handle_mycroft_restart_request(self, message):
             page = join(dirname(__file__), "ui", "Restart.qml")
             self.gui.show_page(page, override_animations=True,
                                override_idle=True)
-        restart_service(self.service_name, sudo=self.use_root)
+        user = False
+        sudo = False
+        # NOTE: Can self.core_service_name be used here?
```

Comment:
yes do this change please

Reply:
Can this detect if it is a user service or a system service?